PyGMO/algorithm/__init__.py

```diff
@@ -1696,3 +1694,23 @@ def _snopt_ctor(
         self.screen_output = screen_output
     snopt._orig_init = snopt.__init__
     snopt.__init__ = _snopt_ctor
+
+# SNOPT algorithm (only if PyGMO has been compiled with the snopt option
```

Comment:
This is correct as it is the snopt wrapper, on the worhp wrapper it does say worhp.